File: Application/Entities/Game.cs
using Domain;

namespace Application.Entities
{
    public class Game
    {
        public Guid Id { get; set; }
        public Guid? FirstPlayerFieldId { get; set; }
        public Guid? SecondPlayerFieldId { get; set; }
        public string? FirstPlayerName { get; set; }
        public string? SecondPlayerName { get; set; }
        public string? NameOfWinner { get; set; }
        public string GameStatus { get; set; }
        public int MoveCount { get; set; }
        public string? ResultInfo { get; set; }
    }
}

File: client-app/src/app/models/game.ts
export interface Game {
    id: string;
    firstPlayerFieldId?: string;
    secondPlayerFieldId?: string;
    firstPlayerName?: string;
    secondPlayerName?: string;
    nameOfWinner?: string;
    status: string;
    moveCount: number;
    resultInfo?: string;
}

export interface FinishGame {
    gameId: string;
    nameOfWinner: string | undefined;
    resultInfo: string;
}